services/community-site/src/pages/Staking/NodeProfilePage.tsx

```diff
@@ -257,14 +261,6 @@ const NodeProfilePage = () => {
                   </div>
                 </>
               )}
-              {/* {validator?.firstBlockCreatedAt && (
-              <>
-                <div className="nodeInfoTitle">node active since</div>
-                <div className="nodeInfoContent">{`${nodeActiveSince.getDate()} ${nodeActiveSince
-                  .toLocaleString('en-US', { month: 'short' })
-                  .toUpperCase()} ${nodeActiveSince.getFullYear().toString().substring(2)}`}</div>
-              </>
-            )} */}
```

Comment:
Let's not remove this. We can add it back now as we have this info in the indexer but should be a new task.